bundles/ui/org.eclipse.smarthome.ui/META-INF/MANIFEST.MF

```diff
@@ -18,6 +18,7 @@ Import-Package: javax.imageio,
  org.eclipse.jetty.client.api,
  org.eclipse.jetty.client.util,
  org.eclipse.jetty.http,
+ org.eclipse.jetty.proxy,
```

Comment:
if you add `optional=true` here, we would be fully backward compatible - i.e. the jetty-proxy bundle does not even have to be present, in which case we would automatically fall back to the old implementation.